packages/core/src/components/Sheet/index.tsx

```diff
@@ -27,6 +29,12 @@ export type Props = {
   gap?: boolean;
   /** Determines if the sheet animates in/out. */
   noAnimation?: boolean;
+  /** Content of the header bar */
+  headerBar?: React.ReactNode;
```

Comment:
@MarkKahn last request/question - what do you think about renaming this to something more generic, like `header`? since the shadow is optional and the header can be anything, more than just a bar